.gitignore

```diff
@@ -1,4 +1,6 @@
 node_modules/
 package-lock.json
 sfdx/yarn.lock
-tmp*/
\ No newline at end of file
+tmp*/
+.idea
+**/oclif.manifest.json
```

Comment:
oclif.manifest.json shouldn't be checked in. It's generated by oclif-dev. See the package.json pre pack phase.